near-contract-standards/src/contract_metadata.rs

```diff
@@ -0,0 +1,13 @@
+/// The contract source metadata is a standard interface that allows auditing and viewing source code for a deployed smart contract.
+#[derive(Debug, serde:: Deserialize, PartialEq)]
```

Comment:
I don't see a reason to restrict users from Clone and serde::Serialize these structs